Lib/typing.py

```diff
@@ -26,7 +25,7 @@
 import contextlib
 import functools
 import operator
-import re as stdlib_re  # Avoid confusion with the re we export.
+import re as stdlib_re  # Avoid confusion with the former re re-export.
```

Comment:
typing.re wasn't a re-export

```suggestion
import re as stdlib_re  # Avoid confusion with the typing.re on <=3.11
```

Reply:
Maybe @srittau wrote it that way just so he could say "re re-export".